herald/src/data/generate-guild-data.js

```diff
@@ -0,0 +1,72 @@
+"use strict"
```

Comment:
Can you put a hyperlink on the home page too (as noted above)?  If not, I can do it later.